### PR TITLE
541 Validate content modification before putting online a page

### PR DESCRIPTION
**Pull request by cmarianhodis**
As per discussed with @clementbonfils ,

When the page status is set to online a check is made if there is any content validation that needs to be done, if so, the validate content pop in will appear on top of the oage validation pop in.

### CODE REVIEW

**Comment by fkroockmann** [comment]
You must validate content when user click on button ok if the page status will be offline.
Can you add a popin to confirm validation of content plz ?

**Comment by cmarianhodis** [comment]
I made the modifications

File: src/tb/component/revisionselector/main.js [hunk 1]
```diff
@@ -116,6 +116,7 @@ define(
                  */
                 save: function () {
                     if (this.config.hasOwnProperty('onSave')) {
+                        sessionStorage.setItem('validateContentFirst', 'false');
```
Comment (fkroockmann) [inline src/tb/component/revisionselector/main.js]: I dont understand why you put that on session storage, why ?

File: src/tb/component/revisionpageselector/main.js [hunk 2]
```diff
@@ -102,13 +103,32 @@ define(
...
+                 * Apply the before save checks to the callback
+                 */
+                beforeSaveChecks: function (changedToOnline) {
+                    var self = this;
+
+                    if (changedToOnline === true) {
+                        ContentRepository.getDrafts().done(function (drafts) {
+                            if (Object.keys(drafts).length > 0) {
+                                sessionStorage.setItem('validateContentFirst', 'true');
+                                Core.ApplicationManager.invokeService('content.main.validate');
```
Comment (fkroockmann) [inline src/tb/component/revisionpageselector/main.js]: that is a promise, you need to wait that promise is done for save

**Comment by fkroockmann** [comment]
I test on local, popin appear for validate content all the time and this popin is too hidden, can you review that ? thanks :)

**Comment by cmarianhodis** [comment]
I made modifications to the code and now when clicking "Yes" in order to validate the content, it will automatically validate the page modifications as well. Also, I moved the content validation pop-in a little lower and a little to the left. Can you please check ?

**Comment by cmarianhodis** [comment]
Solves https://github.com/backbee/backbee-standard/issues/541

Changes made based on:
https://github.com/backbee/backbee-standard/issues/541#issuecomment-182479252

**Pull request merged**